src/helpers/copyFile.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const { terminal } = require('./terminal');
+
+function copyFiles(message) {
+  terminal.question(message, (command) => {
+    const [ commandName, fileToCopy, newFileOrPath ] = command.split(' ');
+    const formatOfNewFile = newFileOrPath.split('.')[1];
+
+    const pathToSourceFile = (`${__dirname}/${fileToCopy}`);
+    const pathToNewFile = formatOfNewFile !== undefined
+      ? (`${__dirname}/${newFileOrPath}`)
+      : (`${newFileOrPath}/${fileToCopy}`);
```

Comment:
You should use `path.join` or `path.resolve` methods

Reply:
**Fixed**